index.js

```diff
@@ -17,7 +17,7 @@ const _spawnArgs = Symbol.for('spawnArgs')
 const nycConfig = process.env.NYC_CONFIG
 
 // the enumerable fields
-const defaults = {
+const defaults = () => ({
```

Comment:
The point of this is to evaluate `process.cwd()` and `Date.now()` when creating a ProcessInfo rather than when istanbul-lib-processinfo is required.  I assume this does not create a performance issue like blindly calling `uuid()` would.  If you disagree I can remove cwd from here and use a conditional `this.cwd = process.cwd()` in the constructor.

Reply:
Makes sense to me.